docs/tasks/capi-operator/add_infrastructure_provider.md

```diff
@@ -1,7 +1,80 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
 ---
 
 # Add Infrastructure Provider
 
-In this section we will cover how to add an infrastructure proviedr such as AWS.
\ No newline at end of file
+This section describes how an infrastructure provider such as Azure could be added using Cluster API Operator.
+
+## Basic Cluster API Provider Installation
+
+This section describes the basic process of installing Cluster API providers using the operator. The Cluster API operator manages five types of objects:
+
+- CoreProvider
+- BootstrapProvider
+- ControlPlaneProvider
+- InfrastructureProvider
+- AddonProvider
+
+Please note that this example provides a basic configuration of Azure Infrastructure provider for getting started. More detailed examples and CRD descriptions will be provided in subsequent sections of this document.
+
+### Installing the CoreProvider
```

Comment:
Is this section really about installing a specific version of the core provider?

Reply:
No not really, addressed this